config/locales/en.yml

```diff
@@ -42,3 +59,23 @@ en:
           friend at %{invitation_page}.
           Cheers, 
           %{archive_name}
+    batch_subscription_notification:
+      subject_linker: "and %{extra} more"
+      chapter_summary: "Chapter Summary: " 
```

Comment:
Is the space after the colon intentional in `"Chapter Summary: "`? I ask because the rest of the strings don't seem to have one.